nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -44,7 +176,11 @@ Maximum Active power|Electricity|Gas|OCGT:
    description: Maximum active power of one typical Open Cycle Gas Turbine power plant
    unit: MW
 
-Maximum Active power|Electricity|Gas|Coal:
+Maximum Active power|Electricity|Gas|OCGT:
+   description: Maximum active power of one typical Combined Cycle Gas Turbine power plant
```

Comment:
This is not the active power of the plant but normalized to the capacity (per MW)

Reply:
this is the complex stuff here. When we model a generation mix that already exist, we can describe power plants one by one and each one has its own technical data (like its maximum active power....) and a name. When we are dealing with the future, what we do is that we describe a serie of  'typical' power plant (because we need to know what is te maximal active power of ONE single plant, as the model will include all the integer constraints....), usually one per technology per country (as countries may manage their plants differently, example: french nuclear plants are flexible and used intensively for providing ancillary services, while in most countries nuclear plants are operated only at nominal power, this can be seen in eg the minimum active power and the duration constraints) and we also describe the total capacity per technology per country. If we did not have the need of describing single units, then the Maximum Active Power variable would not be needed as it would be equal to the capacity.
To answer you comment, no this is not the normalized active power to the capacity (as this would be 1), it really something like 1400MW , 500MW...
Your comments means to me that the description should be improved.
@erikfilias any idea?

Reply:
I totally agree with you, @sandrinecharousset . 
In my point of view, we could make that maximum active power and capacity will be the same because in theory data related to power generation are the same in power and energy models. 

On the other hand, I wrote maximum active power because it is a standard to define the active and reactive power in power system models. Maybe, there will be other models related to AC load flow that use the data or to be included in our project in the future. This is the reason.

In addition, I'm curious about if we will compare the total capacity of power plants used in energy models with the values that we have in power models since the first one usually use a total amount, and the second one use values per power plant, node, region, or country. 

Then, can we conclude about mixing both terms and use only the term capacity?

Reply:
@erikfilias for my model I need both the maximum active power of one plant and the capacity of the technology. Eg I could have for nuclear in france: capacity=50000MW and maximum active power = 1400MW. Thus the model will create the good number of individual units .
then the comparison you suggest in my case is perfectly equivalent :-)

Reply:
Really nice! :)
Then, I will keep the variables unless there is another contrary comment.